docs/cli-guide.rst

```diff
@@ -49,28 +48,31 @@ The plain text file must be plain text encoded in ``UTF-8`` with one
 sentence per line. Paragraph breaks are marked by a blank line, and page
 breaks are marked by two blank lines.
 
-+-----------------------------------+-----------------------------------+
-| Key Options                       | Option descriptions               |
-+===================================+===================================+
-| ``-l, --language`` (required)     | The language code for story.txt.  |
-+-----------------------------------+-----------------------------------+
-| ``-f, --force-overwrite``         | Force overwrite output files      |
-|                                   | (handy if you’re troubleshooting  |
-|                                   | and will be aligning repeatedly)  |
-+-----------------------------------+-----------------------------------+
-| ``-h, --help``                    | Displays CLI guide for            |
-|                                   | ``prepare``                       |
-+-----------------------------------+-----------------------------------+
++-----------------------------------+-----------------------------------------------+
+| Key Options                       | Option descriptions                           |
++===================================+===============================================+
+| ``-l, --language(s)`` (required)  | The language code for story.txt.              |
+|                                   | Specifying multiple comma- or colon-separated |
+|                                   | languages triggers :ref:`g2p-cascade`.        |
++-----------------------------------+-----------------------------------------------+
+| ``-f, --force-overwrite``         | Force overwrite output files                  |
+|                                   | (handy if you're troubleshooting              |
+|                                   | and will be aligning repeatedly)              |
++-----------------------------------+-----------------------------------------------+
+| ``-h, --help``                    | Displays CLI guide for                        |
+|                                   | ``prepare``                                   |
++-----------------------------------+-----------------------------------------------+
 
 The ``-l, --language`` argument requires a language’s 3 character `ISO
 code <https://en.wikipedia.org/wiki/ISO_639-3>`__ as an argument.
 
 The languages supported by RAS can be listed by running ``readalongs prepare -h``
 and they can also be found in the :ref:`cli-prepare` reference.
 
-So, a full command for a story in Algonquin would be something like:
+So, a full command for a story in Algonquin, with a g2p fallback to
+Undetermined, would be something like:
 
-``readalongs prepare -l alq Studio/story.txt Studio/story.xml``
+``readalongs prepare -l alq,und Studio/story.txt Studio/story.xml``
```

Comment:
Although, if we are changing `und` to default anyways, this is redundant.

Reply:
Agreed, it will be. I guess I'll have to review the documentation yet again when I work on that change.

Reply:
done